r/R/dplyr-funcs-datetime.R

```diff
@@ -354,21 +354,21 @@ register_bindings_duration <- function() {
 }
 
 register_bindings_duration_helpers <- function() {
-  register_binding("dseconds", function(x = 1) {
+  make_duration <- function(x, unit) {
```

Comment:
Thanks for this and sorry to be a pain. I think `make_duration()` can be a standalone function, it doesn't need to be a binding since there is no `lubridate` equivalent to it.

Reply:
Oh, my mistake of positioning. Thanks for catching it!

Reply:
Done!